src/main/java/eu/europa/ted/eforms/sdk/domain/Language.java

```diff
@@ -1,53 +1,47 @@
 package eu.europa.ted.eforms.sdk.domain;
 
-import static eu.europa.ec.mdd.generated.Tables.LABEL;
 import java.util.Arrays;
 import java.util.Locale;
-import org.jooq.TableField;
-import eu.europa.ec.mdd.generated.tables.records.LabelRecord;
 
 public enum Language {
 
   // 24 official EU languages.
 
-  BG(new Locale("bg"), LABEL.BG, "Bulgarian", "bul_label"), //
-  CS(new Locale("cs"), LABEL.CS, "Czech", "ces_label"), //
-  DA(new Locale("da"), LABEL.DA, "Danish", "dan_label"), //
-  DE(new Locale("de"), LABEL.DE, "German", "deu_label"), //
-  EL(new Locale("el"), LABEL.EL, "Greek", "ell_label"), //
-
-  EN(new Locale("en"), LABEL.EN, "English", "eng_label"), //
-  ES(new Locale("es"), LABEL.ES, "Spanish", "spa_label"), //
-  ET(new Locale("et"), LABEL.ET, "Estonian", "est_label"), //
-  FI(new Locale("fi"), LABEL.FI, "Finnish", "fin_label"), //
-  FR(new Locale("fr"), LABEL.FR, "French", "fra_label"), //
-
-  GA(new Locale("ga"), LABEL.GA, "Galician", "gle_label"), //
-  HR(new Locale("hr"), LABEL.HR, "Croatian", "hrv_label"), //
-  HU(new Locale("hu"), LABEL.HU, "Hungarian", "hun_label"), //
-  IT(new Locale("it"), LABEL.IT, "Italian", "ita_label"), //
-  LT(new Locale("lt"), LABEL.LT, "Lithuanian", "lit_label"), //
-
-  LV(new Locale("lv"), LABEL.LV, "Latvian", "lav_label"), //
-  MT(new Locale("mt"), LABEL.MT, "Maltese", "mlt_label"), //
-  NL(new Locale("nl"), LABEL.NL, "Dutch", "nld_label"), //
-  PL(new Locale("pl"), LABEL.PL, "Polish", "pol_label"), //
-  PT(new Locale("pt"), LABEL.PT, "Portuguese", "por_label"), //
-
-  RO(new Locale("ro"), LABEL.RO, "Romanian", "ron_label"), //
-  SK(new Locale("sk"), LABEL.SK, "Slovak", "slk_label"), //
-  SL(new Locale("sl"), LABEL.SL, "Slovene", "slv_label"), //
-  SV(new Locale("sv"), LABEL.SV, "Swedish", "swe_label");
+  BG(new Locale("bg"), "Bulgarian", "bul_label"), //
+  CS(new Locale("cs"), "Czech", "ces_label"), //
+  DA(new Locale("da"), "Danish", "dan_label"), //
+  DE(new Locale("de"), "German", "deu_label"), //
+  EL(new Locale("el"), "Greek", "ell_label"), //
+
+  EN(new Locale("en"), "English", "eng_label"), //
+  ES(new Locale("es"), "Spanish", "spa_label"), //
+  ET(new Locale("et"), "Estonian", "est_label"), //
+  FI(new Locale("fi"), "Finnish", "fin_label"), //
+  FR(new Locale("fr"), "French", "fra_label"), //
+
+  GA(new Locale("ga"), "Galician", "gle_label"), //
```

Comment:
This is incorrect, please use the latest version of this from the MDC.

Reply:
Fixed.
The used enum classes have now been copied from MDD and mdd-entities.